build.sbt

```diff
@@ -98,6 +99,7 @@ lazy val allOld = (project in file("allOld"))
   .dependsOn(
     core,
     scalatest,
+    munit,
```

Comment:
`allOld` module is only for backward compatibility. You don't need to add new modules (like `munit`) to this module.